src/dmd/mars.d

```diff
@@ -376,29 +283,17 @@ private int tryMain(size_t argc, const(char)** argv)
 
     if (global.params.mcpuUsage)
     {
-        printf("
-CPU architectures supported by -mcpu=id:
-  =?             list information on all architecture choices
-  =baseline      use default architecture as determined by target
-  =avx           use AVX 1 instructions
-  =avx2          use AVX 2 instructions
-  =native        use CPU architecture that this compiler is running on
-");
+        import dmd.cli : CLIUsage;
+        auto help = CLIUsage.mcpu;
+        printf("%.*s", help.length, &help[0]);
```

Comment:
Is there a reason why `&help[0]` is used instead of `help.ptr`?. `.ptr` is more readable in my opinion.

Reply:
Yes `help.ptr` isn't `@safe`

Reply:
Eh, ok. I didn't know there was a difference.